app/javascript/source/javascript/initializer.js

```diff
@@ -101,6 +101,7 @@ $(document).ready(function() {
       if ($("#admin-content").length > 0) {
         window.binxAdmin = new BinxAdmin();
         binxAdmin.init();
+        binxAdmin.initGraphs();
```

Comment:
you shouldn't add this here, the `init()` method on `binxAdmin` should determine whether something should be called or not (i.e. you should put the `if (window.location.href.match('\\admin/graphs')){` in the init method, and then call `initGraphs()` if it should be.